### PR TITLE
ci: use generate next semver

### PR DESCRIPTION
**Pull request by esoubiran-aneo**
Use the new script to generate next version.

### CODE REVIEW

File: .github/workflows/publish-edge.yml [hunk 1]
```diff
@@ -86,13 +74,16 @@ jobs:
...
-          ref: ${{ github.ref }}
+          fetch-depth: 0
 
       - name: Install .NET Core
         uses: actions/setup-dotnet@v3
         with:
           dotnet-version: 6.x
 
+      - name: Generate Version
+        run: echo "VERSION=$(npx @aneoconsultingfr/generate-next-version@latest --edge)" >> $GITHUB_ENV
```
Comment (aneojgurhem) [inline .github/workflows/publish-edge.yml:85]: on a un paquet head !

Comment (esoubiran-aneo) [reply .github/workflows/publish-edge.yml:86]: @aneojgurhem c'est mis à jour

**Comment by aneojgurhem** [comment]
I'm sure there is more than one ref to fix @esoubiran-aneo .

**Review by aneojgurhem — APPROVED** [review]

**Comment by esoubiran-aneo** [comment]
> I'm sure there is more than one ref to fix @esoubiran-aneo .

Where do I have to fix ref?

**Review by aneojgurhem — APPROVED** [review]

**Comment by esoubiran-aneo** [comment]
Génial, ça fonctionne sur main 3.9.0-edge.42.31bb9ce 🎉

**Pull request merged**